src/partials/options.html

```diff
@@ -1,327 +1,317 @@
+          ng-model="ctrl.panel.statProgressMaxValue"
           ng-change="ctrl.render()"
+          ng-model-onblur
         />
       </div>
-    </div>
-
-    <div class="gf-form" ng-if="ctrl.panel.statProgressType == 'max value'">
-      <label class="gf-form-label width-8">Max Value</label>
-      <input
-        type="number"
-        class="gf-form-input width-14"
-        placeholder="auto" data-placement="right"
-        bs-tooltip="'Max value will be equal to max progreesss in auto mode'"
-        ng-model="ctrl.panel.statProgressMaxValue"
-        ng-change="ctrl.render()"
-        ng-model-onblur
-      />
-    </div>
-
-    <div class="gf-form">
-      <label class="gf-form-label width-8">Null Value</label>
-      <input
-        type="number"
-        class="gf-form-input width-14"
-        placeholder="error" data-placement="right"
-        bs-tooltip="'Map NULL values to constant. Default behaviour is to throw error.'"
-        ng-model="ctrl.panel.nullMapping"
-        ng-change="ctrl.render()"
-        ng-model-onblur
-      />
-    </div>
 
-    <div class="gf-form">
-      <label class="gf-form-label width-8">Mapping type</label>
-      <select
-        class="gf-form-input width-14"
-        ng-model="ctrl.panel.mappingType"
-        ng-options="type for type in ctrl.mappingTypeOptions"
-        ng-change="ctrl.refresh()"
-      />
-    </div>
-
-    <br/>
-    <h5 class="section-heading" ng-if="!ctrl.isMultibar">Tooltip</h5>
-    <div class="gf-form" ng-if="!ctrl.isMultibar">
-      <label class="gf-form-label width-8">Tooltip Mode</label>
-      <div class="gf-form-select-wrapper width-14">
-        <select
-          class="gf-form-input"
-          ng-model="ctrl.panel.tooltipMode"
-          ng-options="n for n in ctrl.tooltipModeOptions"
+      <div class="gf-form">
+        <label class="gf-form-label width-8">Null Value</label>
+        <input
+          type="number"
+          class="gf-form-input width-14"
+          placeholder="error" data-placement="right"
+          bs-tooltip="'Map NULL values to constant. Default behaviour is to throw error.'"
+          ng-model="ctrl.panel.nullMapping"
           ng-change="ctrl.render()"
+          ng-model-onblur
         />
       </div>
-    </div>
 
-    <br/>
-    <h5 class="section-heading">Sorting</h5>
-    <div class="gf-form">
-      <label class="gf-form-label width-8">Type</label>
-      <div class="gf-form-select-wrapper width-14">
+      <div class="gf-form">
+        <label class="gf-form-label width-8">Mapping type</label>
         <select
-          class="gf-form-input"
-          ng-model="ctrl.panel.sortingOrder"
-          ng-options="n for n in ctrl.sortingOrderOptions"
-          ng-change="ctrl.render()"
+          class="gf-form-input width-14"
+          ng-model="ctrl.panel.mappingType"
+          ng-options="type for type in ctrl.mappingTypeOptions"
+          ng-change="ctrl.refresh()"
         />
```

Comment:
can't add a suggestion here, but let's also close the tag properly :)

Reply:
done, thank you